src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -46,21 +46,23 @@
 import java.util.List;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 @Component
 public class JobService {
 
     private static final Logger logger = LoggerFactory.getLogger(JobService.class);
     private static final int MIN_SHUTDOWN_TIMEOUT = 14;
     private static final int POD_LISTENER_SHUTDOWN_TIMEOUT = 2;
+    private static final String API_POD_FILTER = "datarepo-api";
 
     private final Stairway stairway;
     private final IamService samService;
     private final ApplicationConfiguration appConfig;
     private final StairwayJdbcConfiguration stairwayJdbcConfiguration;
     private final MigrateConfiguration migrateConfiguration;
     private final KubeService kubeService;
-    private final JobShutdownState jobShutdownState;
+    private final AtomicBoolean isShutdown;
```

Comment:
I try to avoid booleans that are stated in the negative case. This is positive, but in my mind "shutdown" is itself a negative state, so personally I'd prefer `isRunning`.

Reply:
I get your point. I plan to leave it this way. Otherwise, I get the complaint that isRunning is always inverted because all of the tests are `if (!isRunning)`

Reply:
Ah, but now that I made Shelby's improvement, that is only one simple place, so sure, I'll invert it

Reply:
I'll take it!